tutorials/roofit/rf204a_extendedLikelihood.py

```diff
@@ -0,0 +1,119 @@
+## \file
+## \ingroup tutorial_roofit
+## \notebook
+## Extended maximum likelihood fit in multiple ranges.
+##
+## \macro_code
+##
+
+import ROOT
+
+
+# Setup component pdfs
+# ---------------------
+
+# Declare observable x
+x = ROOT.RooRealVal("x","x",0,11)
+
+# Create two Gaussian PDFs g1(x,mean1,sigma) anf g2(x,mean2,sigma) and their parameters
+mean = ROOT.RooRealVal("mean","mean of gaussians",5)
+sigma1 = ROOT.RooRealVal("sigma1","width of gaussians",0.5)
+sigma2 = ROOT.RooRealVal("sigma2","width of gaussians",1)
```

Comment:
There is a typo here. It should be `RooRealVar` and not `RooRealVal` in lines 16, 19, 20, and 21.